### PR TITLE
Add Genome Nexus version and isoform as comments at top of MAF

### PR DESCRIPTION
**Pull request by mandawilson**
@ao508 @sheridancbio @averyniceday is `isoformOverride` the isoform we wanted to output?

### CODE REVIEW

File: test/data/data_mutations_extended_100.out.uniprot.txt [hunk 1]
```diff
@@ -1,3 +1,5 @@
+#genome_nexus_version: 1.1.1-SNAPSHOT
+#isoform: uniprot
```
Comment (inodb) [inline test/data/data_mutations_extended_100.out.uniprot.txt]: @mandawilson this version will prolly need to be ignored whenever one updates the version so maybe better to not have it in the test output? If you want to test inclusion of the header maybe good to create a separate test that compares the version in the header with the version in the pom instead?

File: test/data/vcf2maf/Makefile [hunk 2]
```diff
@@ -21,7 +21,7 @@ test_output.uniprot.maf:
 	else \
 		AWK=awk; \
 	fi; \
-	(echo $(COMPARE_COLUMNS) | sed 's/\ /	/g'; tail -n+2 $< | $$AWK -vOFS='\t' -vFS='\t' -f ./../../scripts/select_by_column_name.awk $(COMPARE_COLUMNS)) > $@
+	(echo $(COMPARE_COLUMNS) | sed 's/\ /	/g'; grep -v "^#" $< | $$AWK -vOFS='\t' -vFS='\t' -f ./../../scripts/select_by_column_name.awk $(COMPARE_COLUMNS)) > $@
```
Comment (inodb) [inline test/data/vcf2maf/Makefile]: 🆗

File: test/data/data_mutations_extended_100.out.uniprot.txt (hunk 1 above)
Comment (mandawilson) [reply test/data/data_mutations_extended_100.out.uniprot.txt]: I see what you mean.  I just removed it but we need to then change this circle ci test: ```java -jar annotationPipeline/target/annotationPipeline-*.jar --filename test/data/data_mutations_extended_100.in.txt  --output-filename test/data/data_mutations_extended_100.out.uniprot.txt --isoform-override uniprot && git diff --exit-code test/data/data_mutations_extended_100.out.uniprot.txt || (echo -e "MAF uniprot output changed test/data/data_mutations_extended_100.out.uniprot.txt $(cat test/data/data_mutations_extended_100.out.uniprot.txt | curl -F c=@- https://ptpb.pw | grep url) (include this file in your PR to fix this test)" && exit 1)``` We should probably write to a different file, and then do an ordinary diff that cuts out any comments.  What do you think @inodb?

Comment (mandawilson) [reply test/data/data_mutations_extended_100.out.uniprot.txt]: @inodb Actually I think I can change the git diff, and then it should work.

**Pull request merged**